### ISSUE TITLE
Seriously what is github turning into ?

### ISSUE THREAD

**Issue by dsslimshaddy** [bachkukkik/awesome-thai-girls#9]
This needs to like stop before it gets out of hand

**Comment by xingmarc** [comment]
Well I was thinking this list is generated by some fancy crawlers based on some interesting algorithms, but i was wrong...so wrong ...

**Comment by dsslimshaddy** [comment]
yeah if it had some like face detection cuteness percent algorithm then its something
